src/Raven.Client/Documents/Session/DocumentsChanges.cs

```diff
@@ -17,11 +18,13 @@ public class DocumentsChanges
         /// <summary>
         /// Previous field type.
         /// </summary>
+        [Obsolete("No longer used")]
         public BlittableJsonToken FieldOldType { get; set; }
 
         /// <summary>
         /// Current field type.
         /// </summary>
+        [Obsolete("No longer used")]
```

Comment:
`[Obsolete("DocumentsChanges.FieldNewType is not supported anymore. Will be removed in next major version of the product.")]`